model/model_training/configs/config.yaml

```diff
@@ -149,6 +162,7 @@ oasst_export_eu:
   use_custom_sampler: false
```

Comment:
this configuration is a bit strange (I know .. you didn't touch it) but I don't think it was ever used but for one test run .. e.g. it consists mainly of gpt4all with alone is >400k samples ... if it should be used for finetuning it would probably good to balance the datasets with the custom-sampler ...

Reply:
i don't know how exactly the custom sampler works, so i'm not going to touch it for now..